src/wayback/exceptions.py

```diff
@@ -101,8 +101,8 @@ def __init__(self, response, retry_after):
         super().__init__(message)
 
 
-class SessionClosedError(Exception):
+class AlreadyClosedError(Exception):
     """
-    Raised when a Wayback session is used to make a request after it has been
+    Raised when a Wayback client is used to make a request after it has been
```

Comment:
It didn’t make sense to keep “session” in the name, but instead of replacing it with “adapter” I decided to go with making it a little more generic.